Include/modsupport.h

```diff
@@ -139,6 +139,9 @@ void _PyArg_Fini(void);
 PyAPI_FUNC(int) PyModule_AddObject(PyObject *, const char *, PyObject *);
 PyAPI_FUNC(int) PyModule_AddIntConstant(PyObject *, const char *, long);
 PyAPI_FUNC(int) PyModule_AddStringConstant(PyObject *, const char *, const char *);
+#ifndef Py_LIMITED_API
+PyAPI_FUNC(int) _PyModule_AddType(PyObject *module, PyTypeObject *type);
```

Comment:
IMO _PyModule_AddType deserves to be added to the limited C API and be documented. You can keep the declaration in this file but use:

```
#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x03090000
/* New in 3.9 */
(...)
#endif
```

Add the doc to Doc/c-api/module.rst, near PyModule_AddObject().